### DISCUSSION TITLE
Dell U2720Q audio volume control not working?

### DISCUSSION THREAD

**Discussion by BermudaBen** [MonitorControl/MonitorControl#1191] [General]
My Dell U2720Q is connected via USB-C to my MacBook Pro M1. External speakers are connected to the Dell via audio line-out port. I can control screen brightness using an Apple keyboard with the MonitorControl app but not audio volume. Audio is output at full volume through the speakers.

Can anyone confirm that the Dell U2720Q cannot be properly controlled with this app? Or can anyone point me to a solution?

**Comment by user858753257** [comment]
I have a similiar problem with my dell S3224DWC 

Maybe DDC supports no audio control

**Reply by paulnaslund** [reply]
Having the same issue with my Dell U2720Q/2021 MBP. I have brightness controls but no audio volume.
